.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,34 @@
+name: "🔍 Potential Duplicates"
+on:
+  issues:
+    types: [opened, edited] # edited means the issue title changed
```

Comment:
For manual start

```suggestion
on:
  issues:
    types: [opened, edited] # edited means the issue title changed
  workflow_dispatch:
```

Reply:
A manual start won't work, because this action is issue related. It's not running over all issues and checks the duplicates on a manual start. This action requires the creation or edit event of an issue to work properly:

https://github.com/wow-actions/potential-duplicates/blob/master/src/action.ts#L14

Reply:
Ok